### PR TITLE
Completed dom-querying| Mohamed Elmessiry

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-02-01 at 2 25 52 PM

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+console.log('hello, world');
+var q = document.querySelector('h1');
+console.log('h1 query selectors:' + ' ' + q);
```
Comment (thebearingedge) [inline dom-querying/main.js:3]: For your `console.log`, use two arguments instead of one concatenated string.

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+console.dir('h1 directory query' + ' ' + q);
+var p = document.querySelector('#explanation');
+console.log(' #explanation id query selector' + ' ' + p);
+console.dir('p directory query ' + ' ' + p);
+var f = document.querySelector('.hint');
+console.log('hint class query selector ' + ' ' + f);
+console.dir('hint class query directory' + ' ' + f);
```
Comment (thebearingedge) [inline dom-querying/main.js:10]: For `console.dir`, only pass the variable with no label.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please also update your screenshot when the changes have been made. Thank you!

File: dom-querying/main.js (hunk 2 above)
Comment (Mohamed-Elmessiry) [reply dom-querying/main.js]: Screen Shot 2021-02-02 at 11 00 35 AM
Screen Shot 2021-02-02 at 11 00 43 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request closed without merging**